f/respawn/respawn_settings.sqf

```diff
@@ -76,7 +80,20 @@ respawnMenuRoles = [
   ["C", "Crewman - Vehicle Crew"],
   ["D", "Crewman - Vehicle Driver"],
   ["PP", "Aircrewman - Pilot"],
-  ["PCC", "Aircrewman - Crew Chief (Repair)"]
+  ["PCC", "Aircrewman - Crew Chief (Repair)"],
+  ["VC","Vehicle Commander"],
```

Comment:
Backfill current factions to use the normal "Vehicle Crew" when this option is selected, so if someone choses this for any other factions it doesn't blow up.